### PR TITLE
CSAM Reports in Case Print View

### PR DESCRIPTION
**Pull request by mythilytm**
Primary reviewer:

## Description
- This PR adds the CSAM Reporting creation date and reference number to the Case Print PDF view.
- Added styles conforming to current structure for CSAM reports similar to summary styling

### Checklist
- [x] Corresponding issue has been opened
- [N/A] New tests added
- [N/A] Feature flags added
- [N/A] Strings are localized
- [N/A] Tested for chat contacts
- [N/A] Tested for call contacts

### Related Issues
Fixes #  [CHI-2092](https://tech-matters.atlassian.net/browse/CHI-2092)

### Verification steps
1. Open a pdf for a case that has CSAM reports associated with it. 
CSAM - PDF view

[CHI-2092]: https://tech-matters.atlassian.net/browse/CHI-2092?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: plugin-hrm-form/src/components/case/casePrint/CasePrintView.tsx [hunk 1]
```diff
@@ -171,6 +172,7 @@ const CasePrintView: React.FC<Props> = ({ onClickClose, caseDetails, definitionV
                     values={{
                       ...caseDetails.contact?.rawJson?.caseInformation,
                       ...caseDetails.contact?.rawJson?.childInformation,
+                      ...caseDetails.contact?.csamReports,
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/case/casePrint/CasePrintView.tsx:175]: Is this needed if you are also rendering `<CasePrintCSAMReports csamReports={caseDetails.contact?.csamReports} />` at the bottom of this file?

File: plugin-hrm-form/src/components/case/casePrint/CasePrintCSAMReports.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+/* eslint-disable dot-notation */
+import React from 'react';
+import { Text, View } from '@react-pdf/renderer';
+import { format } from 'date-fns';
+
+import styles from './styles';
+import { getTemplateStrings } from '../../../hrmConfig';
+
+type OwnProps = {
+  csamReports?: Array<any>;
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/case/casePrint/CasePrintCSAMReports.tsx:28]: Could you add a stronger typing here? I think we have the CSAM type somewhere.

File: plugin-hrm-form/src/components/case/casePrint/CasePrintCSAMReports.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            <Text>{strings['CSAMReportForm-Self-Attachment']}</Text>
+          );
+
+        const date = `${format(new Date(csamReport.createdAt), 'yyyy-MM dd h:mm aaaaa')}m`;
+        return (
+          <View style={styles['sectionBody']} key={index}>
+            <Text style={styles['csamReportText']}>{template}</Text>
+            <Text style={styles['csamReportText']}>{date}</Text>
+            <Text style={styles['csamReportText']}>{`#${csamReport.csamReportId}`}</Text>
+          </View>
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/case/casePrint/CasePrintCSAMReports.tsx:55]: I know this has been reviewed by the product team and they said is fine. I would suggest prepending something like a short description (`date`,  `id`, etc) to each line, but that's totally a personal opinion and you can ignore me and favor whatever the product team is fine with.

**Review by GPaoloni — APPROVED** [review]
Small comments but nothing blocking. LGTM!

File: plugin-hrm-form/src/components/case/casePrint/CasePrintView.tsx (hunk 1 above)
Comment (mythilytm) [reply plugin-hrm-form/src/components/case/casePrint/CasePrintView.tsx:175]: You are right. Not needed, I was supposed to remove this.

File: plugin-hrm-form/src/components/case/casePrint/CasePrintCSAMReports.tsx (hunk 3 above)
Comment (mythilytm) [reply plugin-hrm-form/src/components/case/casePrint/CasePrintCSAMReports.tsx:55]: I will bring this up via qa notes.

File: plugin-hrm-form/src/components/case/casePrint/CasePrintCSAMReports.tsx (hunk 2 above)
Comment (mythilytm) [reply plugin-hrm-form/src/components/case/casePrint/CasePrintCSAMReports.tsx:28]: Okay.

**Pull request merged**